docs/index.txt

```diff
@@ -7,64 +7,111 @@ Laravel MongoDB
 
-This package was renamed to ``mongodb/laravel-mongodb`` because of a transfer
-of ownership to MongoDB, Inc. It is compatible with Laravel 10.x. For older
-versions of Laravel, please see the `old versions <https://github.com/mongodb/laravel-mongodb/tree/3.9#laravel-version-compatibility>`__.
+Fundamentals
+------------
+
+To learn how to perform the following tasks by using the {+odm-short+},
+see the following content:
 
-- :ref:`laravel-install`
 - :ref:`laravel-eloquent-models`
 - :ref:`laravel-query-builder`
 - :ref:`laravel-user-authentication`
 - :ref:`laravel-queues`
 - :ref:`laravel-transactions`
-- :ref:`laravel-upgrading`
+
+Upgrade Versions
+----------------
+
+Learn what changes you might need to make to your application to upgrade
+versions in the :ref:`laravel-upgrading` section.
 
 Reporting Issues
 ----------------
 
-Think you’ve found a bug in the library? Want to see a new feature? Please open a case in our issue management tool, JIRA:
+We are lucky to have a vibrant PHP community that includes users of varying
+experience with MongoDB PHP Library and {+odm-short+}. To get support for
+general questions, search or post in the
+`MongoDB Community Forums <https://www.mongodb.com/community/forums/>`__.
 
-- `Create an account and login <https://jira.mongodb.org/>`__
-- Navigate to the `PHPORM <https://jira.mongodb.org/browse/PHPORM>`__ project.
-- Click Create
-- Please provide as much information as possible about the issue type and how to reproduce it.
+To learn more about MongoDB support options, see the
+`Technical Support <https://www.mongodb.org/about/support>`__ page.
 
-Note: All reported issues in JIRA project are public.
 
-For general questions and support requests, please use one of MongoDB's 
-:manual:`Technical Support </support/>` channels.
+Bugs / Feature Requests
+-----------------------
 
-Security Vulnerabilities
-~~~~~~~~~~~~~~~~~~~~~~~~
+If you've found a bug or want to see a new feature in {+odm-short+},
+please report it in the GitHub issues section of the
+`mongodb/laravel-mongodb <https://github.com/mongodb/laravel-mongodb>`__
+repository.
 
-If you've identified a security vulnerability in a driver or any other MongoDB 
-project, please report it according to the instructions in 
-:manual:`Create a Vulnerability Report </tutorial/create-a-vulnerability-report>`.
+To report a bug or request a new feature, perform the following steps:
 
+1. Visit the `GitHub issues <https://github.com/mongodb/laravel-mongodb/issues>`__
+   section and search for any similar issues or bugs.
+#. If you find a matching issue, you can reply to the thread to report that
+   you have a similar issue or request.
+#. If you cannot find a matching issue, click :guilabel:`New issue` and select
+   the appropriate issue type.
+#. If you selected "Bug report" or "Feature request", please provide as much
+   information as possible about the issue. Click :guilabel:`Submit new issue`
+   to complete your submission.
```

Comment:
We may indicate that if they know how to fix the issue, they can create a PR. With a link to the PR section.